domain/application/state/types.go

```diff
@@ -876,3 +877,8 @@ func decodeWorkloadStatus(s int) (application.WorkloadStatusType, error) {
 		return -1, errors.Errorf("unknown status %d", s)
 	}
 }
+
+type minimalStorage struct {
```

Comment:
minimalStorage to me doesn't tell me what  this thing is or does. Just a nit but it sounds like a Juju concept on first read that I didn't know about.

Reply:
What about storageIdMapping ?

Reply:
We use the "minimal" naming patter elsewhere as well.
In the next PR, life and other attributes will be added, but not all of them, just a "minimal" set.
I'll rename this one to "storageDetails" and we can see what people like best and adopt a consistent approach.